kotest-framework/kotest-framework-engine/src/jvmMain/kotlin/io/kotest/engine/concurrency/FixedThreadCoroutineDispatcherFactory.kt

```diff
@@ -31,22 +32,27 @@ import kotlin.reflect.KClass
 class FixedThreadCoroutineDispatcherFactory(
    threads: Int,
    private val affinity: Boolean,
-) : CoroutineDispatcherFactory {
+) : CoroutineDispatcherFactory, Closeable {
```

Comment:
Are the changes in this class mostly nomenclature ?

Reply:
Not really. The `close()` is essential to help avoid leaking threads. The rest is (hopefully) about communicating intent (needed this for myself to better understand things).

Reply:
So in the whole class, it's close + nomenclature ?

Reply:
Yes, plus using a kotlinx.coroutines dispatcher instead of Executor.